modules/checking-mco-status.adoc

```diff
@@ -15,11 +15,36 @@ To see the status of the Machine Config Operator, its sub-components, and the re
 ----
 $ oc get machineconfigpool
 NAME      CONFIG                  UPDATED  UPDATING   DEGRADED  MACHINECOUNT  READYMACHINECOUNT  UPDATEDMACHINECOUNT DEGRADEDMACHINECOUNT  AGE
-master    rendered-master-dd…     True     False      False     3             3                  3                                0                     4h42m
-worker    rendered-worker-fde…    True     False      False     3             3                  3                                0                     4h42m
+master    rendered-master-dd…     True     False      False     3             3                  3                                0        4h42m
+worker    rendered-worker-fde…    True     False      False     3             3                  3                                0        4h42m
 ----
 +
+where:
++
+--
+** `UPDATED` indicates that the MCO applied the current machine config to the nodes in that MCP.
+** `UPDATING` indicates that the MCO is applying the current machine config to at least one of the nodes in that MCP. Nodes that are in `UPDATING` status might not available for scheduling.
```

Comment:
current machine config should be desired machine config at `spec.configuration.name` of mcp, current config here means value of column config in output of `oc get mcp`. it can be found at `status.configuration.name`

Node does not have `UPDATING` status. it could be `NotReady` and `SchedulingDisabled`